code/src/Album.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+// import data from './data.json'
+
+import "./components/album.css";
+
+export const Album = (props) => {
+  return (
+    <section className="album-section">
+      <div>
+        <div className="cover">
+        <div className="icons">
+        <img className="heart" src="./icons/heart.svg" alt="Heart" />
+        <a href={props.albumURL} alt="songURL"><img className="play" src="./icons/play.svg" alt="play" /></a>
+        <img className="dots" src="./icons/dots.svg" alt="dots" />
+        </div>
+        <img className="cover-img" src={props.cover} alt="coverImg" />
+        </div>
+        <div className="album-name">
+          <a href={props.albumURL}>{props.name}</a>
+        
+        </div>
+        <div className="artist-name">
+        <a href={props.artistURL}>{props.artist}</a>
```

Comment:
Here, to be able to show multiple artists (if there are multiple artists on the album) do a similar solution as you did in App.js to get all the albums. 
```
{props.artists.map(artist => (
    <a className="artist" href={artist.external_urls.spotify}>{artist.name}</a>
))}
```